ui/values/localizable/en.go

```diff
@@ -923,13 +924,16 @@ const EN = `
 "updateDEXWalletPasswordReason" = "Your %s wallet (%s) is connected to your DEX account. Please provide your DEX password to update DEX with the new wallet password."
 "loginDEXForActiveOrdersOrExpiredBonds" = "You have one or more active orders and/or expired bonds waiting to be refunded, login to DEX now!"
 "walletsNeedToSync" = "Wallet(s) need to sync"
-"walletsNeedToSyncMsg" = "Wallet(s) used by the DEX client needs to sync to settle active trade(s) or refund expired bonds."
+"walletsNeedToSyncMsg" = "The following wallet(s) used by the DEX client needs to sync to settle active trade(s) or refund expired bonds: %s"
 "iWillSyncLater" = "I'll sync later"
+"iWillLoginLater" = "I'll login later"
 "okaySync" = "Okay, sync"
 "paste" = "Paste"
 "backupDEXSeed" = "Backup DEX Seed"
 "dexSeed" = "DEX Seed"
 "optionalRestorationSeed" = "Restoration Seed (optional)"
+"dexServerDisconnected" = "DEX server is disconnected"
+"dexMsgAfterReset" = "It seems DCRDEX has been reset, please revisit this page."
```

Comment:
I'm not sure ordinary users will understand how to "revisit" a page. Why can't we reopen the page for them automatically?

Reply:
> Why can't we reopen the page for them automatically? 

The page needs an initialized dexc client which we only prepare in the trade page. If the last page on the stack after the user resets dex was the dex page, we might not have an initialized client by the time they close the settings page and go back to the dex page.

But, I think we can show the splash page instead until dexc is ready.